docs/smoothing.rst

```diff
@@ -72,6 +72,28 @@ the error persists, to then slowly increase epsilon until a valid common beam is
 found. More information can be found in the
 `radio-beam documentation <https://radio-beam.readthedocs.io/en/latest/>`_.
 
+Alternative approach to spatial smoothing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There is an alternative way to spatially smooth a data cube, which is using the
+:meth:`~spectral_cube.SpectralCube.spatial_smooth` method. Example code snippets
+are as follows::
+
+    from spectral_cube import SpectralCube
+    from astropy.io import fits
+    from astropy.convolution import Gaussian2DKernel
+
+    cube0 = fits.open('/some_path/some_file.fits')
+    cube = SpectralCube.read(cube0)
+    kernel = Gaussian2DKernel(1)
```

Comment:
use keyword arguments here, e.g. `x_stddev=1`.  Then you don't need to say nearly as much below